src/assessmentQuestion.ts

```diff
@@ -701,6 +701,8 @@ export const AssessmentSyncModel = makeEnum({
   BusinessEntity: 'businessEntity',
   /** ProcessingActivity model */
   ProcessingActivity: 'processingActivity',
+  /** SensitiveCategory model */
+  SensitiveCategory: 'sensitiveCategory',
```

Comment:
I don't think we need this as a sync model. We need to be able to sync a sensitive category back to dataSubCategories, but we don't need to sync any fields back to sensitive categories.